troubleshooting.adoc

```diff
@@ -228,6 +228,16 @@ image::./images/github_org_error.png["GitHub authentication error - user not in
 * Check that the user attempting login is a member of at least one of the GitHub organizations specified in the AuthRealm. 
 * If the GitHub OAuth application is not owned by an organization specified in `github.organizations`, an organization owner must grant third-party access to use this option. 
 
+=== Internal Server Error: missing access_token
+
+If after you press the OpenShift login button, you see a page displaying "Internal Server Error" and "Failed to authenticate: github: failed to get token: oauth2: server response missing access_token", you likely have an invalid GitHub OAuth client id and secret in your AuthRealm. Review your AuthRealm CR `github.clientID` and `github.clientSecret` fields, and ensure the base64-encoded client secret is correct by running the following:
+[source,terminal]
+----
+$ oc get AuthRealm <your_authrealm_name> -n <your_authrealm_namespace> -ojson | jq -r '.spec.identityProviders[0].github'
```

Comment:
In rare case, If they have more than one identity providers in authrealm. This command will fail to get secret name

Reply:
I think we can just mention, get the secret name from spec.identityProviders.github.clientSecret in authrealm

Reply:
Thanks, @leena-jawale ! Please take a look at the changes I've pushed.